_commontech/Customer_3.1.0_Release_Notes.md

```diff
@@ -42,18 +42,18 @@ Release 3.1.0 of **IBM Blue Box Cloud, Dedicated and Local**, includes some new
 3.) **Storage Performance Improvements**: **Ceph** deployments have been upgraded to the Jewel release with Jemalloc enabled, to provide a performance improvement.
 
 
-**IBM Blue Box Cloud Image Updates**
+**IBM Bluemix on OpenStack Image Updates**
 
-**An update for IBM Blue Box Cloud Images**: Our library of available Cloud Images has been updated. The [Cloud Image User Guide]( http://ibm-blue-box-help.github.io/help-documentation/gettingstarted/userguides/cloud_images/image_patch_list_20160910/Image_Release_Notes_2016-09-10/) has the latest information. 
+**An update for IBM Bluemix on OpenStack Images**: Our library of available Cloud Images has been updated. The [Cloud Image User Guide]( http://ibm-blue-box-help.github.io/help-documentation/gettingstarted/userguides/cloud_images/image_patch_list_20160910/Image_Release_Notes_2016-09-10/) has the latest information. 
 
-**IBM Blue Box 3.1.0 Dedicated Cloud features:**
+**IBM Bluemix Dedicated on OpenStack 3.1.0 features:**
 
- * Blue Box Dedicated is now ISO27001 and 270018 compliant, plus SOC 2 and 3.
+ * Bluemix Dedicated on OpenStack is now ISO27001 and 270018 compliant, plus SOC 2 and 3.
 
-**IBM Blue Box 3.1.0 Local Cloud features:**
+**IBM Bluemix Local on OpenStack 3.1.0 features:**
 
  * **Support for Lenovo**: In addition to the current Cisco UCS platform, a Lenovo-curated bill of materials is now supported.  
- * **Support for Bluemix Local System**: IBM Blue Box Local (PCaaS) Private Cloud as a Service is now supported on Bluemix Local Systems.
+ * **Support for Bluemix Local System**: IBM Bluemix Local on OpenStack (PCaaS) Private Cloud as a Service is now supported on Bluemix Local Systems.
```

Comment:
Note that Bluemix Local Systems is the new name for PureApp. The wording of this sentence was given to me directly by Offering Management. No change is needed here, I just wanted to highlight this information for you about the name change for PureApp. Thanks!